packages/pwa/app/utils/routes-utils.test.js

```diff
@@ -7,40 +7,228 @@
 import React from 'react'
 
 import {configureRoutes} from './routes-utils'
-import {getUrlConfig} from './utils'
-
-jest.mock('./utils', () => {
-    const original = jest.requireActual('./utils')
-    return {
-        ...original,
-        getUrlConfig: jest.fn()
-    }
-})
 
 describe('configureRoutes', function() {
-    test('should only configured the routes with :locale as part of the path', () => {
-        getUrlConfig.mockImplementation(() => ({
-            locale: 'path'
-        }))
-        const CompA = () => <div>This is component A</div>
-        const CompB = () => <div>This is component B</div>
+    const cases = [
+        {
+            urlConfig: {
+                site: 'path',
+                locale: 'path',
+                showDefaults: true
+            },
+            expectedRes: [
+                '/uk/en-uk/',
```

Comment:
Thanks for writing all of these routes tests. They give a boost of confidence that this complex function works as expected 👍